docs/guides/support.md

```diff
@@ -346,16 +346,12 @@ business impact that the feature will bring with future research done.
 
 ### Severity
 
-There are 4 levels of severity for bugs in the Carbon core repositories:
-
-- https://github.com/carbon-design-system/carbon/labels/severity%3A%201 - _"Must
-  be fixed ASAP"_
-- https://github.com/carbon-design-system/carbon/labels/severity%3A%202 - _"User
-  cannot complete task, and/or no workaround"_
-- https://github.com/carbon-design-system/carbon/labels/severity%3A%203 - _"User
-  can complete task, and/or has a workaround"_
-- https://github.com/carbon-design-system/carbon/labels/severity%3A%204 -
-  _"Unrelated to a user task, has a workaround or does not need a workaround"_
+|                                   Level                                    |                                                             Description                                                             | Kit                                                                                                              | Docs                                                                                                                                           | Code                                                                                                             | Release                                                                                                                                                                          |
+| :------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------------------: | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [1](https://github.com/carbon-design-system/carbon/labels/severity%3A%201) | Affects major functionality. Blocks users from completing tasks, exposes intellectual property or damages the brand. No workaround. | The kit is has major functionality defects, designers can’t design and there is no work-around                   | The documentation damages the brand or exposes intellectual property.                                                                          | The code has major functionality defects, breaking builds and can’t be worked around.                            | If the problem is in production, it must be fixed as soon as possible in a patch.                                                                                                |
```

Comment:
If you remove the markdown formatting around these links GitHub will display the actual label itself. It's still a link, too.

```suggestion
| https://github.com/carbon-design-system/carbon/labels/severity%3A%201 | Affects major functionality. Blocks users from completing tasks, exposes intellectual property or damages the brand. No workaround. | The kit is has major functionality defects, designers can’t design and there is no work-around                   | The documentation damages the brand or exposes intellectual property.                                                                          | The code has major functionality defects, breaking builds and can’t be worked around.                            | If the problem is in production, it must be fixed as soon as possible in a patch.                                                                                                |
```